### PR TITLE
#33 Added configurable idle time for a SaaS database

### LINKED ISSUE #33: Make the saas_database fixture configurable in terms of the idle time
A user must be able to inject the desired idle time, to be used as the `idle_time` parameter when calling the `api_access.database(...)`.

### PR DESCRIPTION
**Pull request by ahsimb**
closes #33

### CODE REVIEW

File: pytest-saas/exasol/pytest_saas/__init__.py [hunk 1]
```diff
@@ -33,6 +33,16 @@ def pytest_addoption(parser):
             pytest plugin for exasol-saas-api will include this short tag into
             the names of created database instances.""",
     )
+    parser.addoption(
+        "--idle-time",
+        action="store",
+        default="2",
```
Comment (tomuben) [inline pytest-saas/exasol/pytest_saas/__init__.py:39]: This mean we could have a minimum idle time of 1h. Couldn't there be cases where we want to have a shorter idle-time, maybe 30min or so?

Comment (ahsimb) [reply pytest-saas/exasol/pytest_saas/__init__.py:39]: Firstly, it's a float, 30 min is 0.5.
Secondly, I don't see cases when we need to set it to a shot period. The database fixture is supposed to shut down the database as soon as all tests are done. Unless we want to keep the database to investigate the effect of our tests. In that case, we probably want to give ourselves enough time for investigation and then shut down the db manually.

Comment (ckunki) [reply pytest-saas/exasol/pytest_saas/__init__.py:39]: I am wondering about whether we should rename the CLI option.
I think `--idle-time` is very unspecific, it should say something about the semantics of the idle duration:
* which component is affected, in this case is clear I think: the SaaS databse instance 
* what will happen after the timeout, i.e. an autostop 
* especially the unit should be clear, AFAIK @ahsimb chose hours which also deviates from the unit _minutes_ in the original SaaS API.

In [saas-api-python/.../__init__.py](https://github.com/exasol/saas-api-python/blob/main/exasol/saas/client/__init__.py) the constant is called `AUTOSTOP_DEFAULT_IDLE_TIME` with datatype `timedelta`.

**Review by ckunki — CHANGES_REQUESTED** [review]

**Comment by ahsimb** [comment]
@ckunki, how about
`--saas-timeout`
or
`--saas-idle-hours`

Hours are better than minutes (which, in turn, better than seconds and milliseconds).

**Comment by ahsimb** [comment]
OK, the ticket got more complicated than originally thought. Back to the drawing board.

**Comment by ahsimb** [comment]
Please ignore my previous comment. I decided to do the follow-up development in a separate PR.

**Review by tomuben — APPROVED** [review]

**Review by ckunki — APPROVED** [review]

**Pull request merged**